src/entities/media-entry.js

```diff
@@ -35,6 +35,11 @@ export default class MediaEntry {
    * @type {string}
    */
   type: string;
+  /**
+   * @member - entry mediaType
+   * @type {string}
+   */
+  mediaType: string;
```

Comment:
we try to keep the base class free of OVP/OTT so this changes the response object, although you did make it optional in media config source.
In general, unless object should be partial, it is not advised to use optionals in it(leads to many null|undefined checks everywhere).
Any way to do this better?
If optional I would use the metadata container which is exactly that.

Reply:
you're right but we didn't use mediaEntry as is, we're parsing mediaEntry in _getSourcesObject mediaEntry to sourceObj

Reply:
yes, it's the one above when doing full code review, but still, would like to avoid going to situation where most of the members in `ProviderMediaConfigSourcesObject` are optional, and we have too many already now